third_party/vmServiceDrivers/org/dartlang/vm/service/VmServiceBase.java

```diff
@@ -268,8 +258,9 @@ public void onError(RPCError error) {
       @Override
       public void received(Obj response) {
         if (response instanceof Instance) {
-          consumer.received((Instance) response);
-        } else {
+          consumer.received((Instance)response);
+        }
```

Comment:
nit: seems like surprising formatting. has the formatter been run?

Reply:
This is straight from the generated code.

Reply:
Re-generation does correct the formatting here, so I might have unintentionally formatted this -